compose/test.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.6'
+
+services:
+
+  terraform-provider-logdna:
+    build:
+      context: ../
+      dockerfile: Dockerfile
+      args:
+        - GO_VERSION
+    volumes:
+      - ../coverage:/opt/app/coverage
+    environment:
+      - servicekey
+      - COVERALLS_TOKEN
```

Comment:
newline here

also I think it might be good to rename the `servicekey` env var here and in the code, it's only used for tests. `SERVICE_KEY` feels appropriate.

Reply:
I definitely had that ear marked to change everywhere, including in the code.  I'll fix in another commit on this PR.